docs/installation.rst

```diff
@@ -22,7 +22,19 @@ Pre-build package available at PyPi:
 
     pip install highdicom
 
-Source code available at Github:
+Like the underlying ``pydicom`` package, highdicom relies on functionality
+implemented in the ``pylibjpeg-libjpeg``
+`package <https://pypi.org/project/pylibjpeg-libjpeg/>`_ for the decoding of
+DICOM images with certain transfer syntaxes. Since ``pylibjpeg-libjpeg`` is
+licensed under a copyleft GPL v3 license, it is not installed by default when
+you install highdicom. To install ``pylibjpeg-libjpeg`` along with highdicom,
+use
```

Comment:
```suggestion
The library relies on the underlying ``pydicom`` package for decoding of pixel data, which internally delegates the task to either the ``pillow`` or the ``pylibjpeg`` packages.
Since the ``pillow`` is a dependency of *highdicom* and will automatically be installed, some transfer syntax can thus be readily decoded and encoded (baseline JPEG, JPEG-2000, JPEG-LS).
Support for additional transfer syntaxes (e.g., lossless JPEG) requires installation of the ``pylibjpeg`` package as well as the ``pylibjpeg-libjpeg`` and ``pylibjpeg-openjpeg`` packages.
Since ``pylibjpeg-libjpeg`` is licensed under a copyleft GPL v3 license, it is not installed by default when you install *highdicom*. To install the ``pylibjpeg`` packages along with *highdicom*, use
```